<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectEditorImage</class>
 <widget class="QWidget" name="ObjectEditorImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>89</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Size:</string>
        </property>
        <property name="buddy">
         <cstring>widthSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QToolButton" name="flipButton">
        <property name="toolTip">
         <string>Flip</string>
        </property>
        <property name="text">
         <string>Flip</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QCheckBox" name="constrainRatioCheckBox">
        <property name="text">
         <string>Constrain ra&amp;tio</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QToolButton" name="rotateRightButton">
        <property name="toolTip">
         <string>Rotate right</string>
        </property>
        <property name="text">
         <string>Rotate right</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QToolButton" name="rotateLeftButton">
        <property name="toolTip">
         <string>Rotate left</string>
        </property>
        <property name="text">
         <string>Rotate left</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>C&amp;lose</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <spacer name="horizontal">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>93</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>78</width>
          <height>1</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7">
       <spacer name="horizontal_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="widthSpinBox">
        <property name="toolTip">
         <string>Width</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="heightSpinBox">
        <property name="toolTip">
         <string>Height</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="resizeButton">
        <property name="text">
         <string>&amp;Resize</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontal_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>27</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="10">
       <widget class="QToolButton" name="mirrorButton">
        <property name="toolTip">
         <string>Mirror</string>
        </property>
        <property name="text">
         <string>Mirror</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>rotateRightButton</tabstop>
  <tabstop>rotateLeftButton</tabstop>
  <tabstop>mirrorButton</tabstop>
  <tabstop>flipButton</tabstop>
 </tabstops>
 <connections/>
</ui>
